lib/busbar_cli/models/nodetype.rb

```diff
@@ -0,0 +1,12 @@
+class NodeType
+  include Virtus.model
+
+  attribute :id, String
+  attribute :cpu, String
+  attribute :guaranteed_cpu, String
+  attribute :memory, String
+
+  def as_text
+    "#{id} =>\tCPU: #{cpu} \tMemory: #{memory}"
```

Comment:
Must there be a space before the tab?
```suggestion
    "#{id} =>\tCPU: #{cpu}\tMemory: #{memory}"
```